index.js

```diff
@@ -213,10 +277,19 @@ const argTimesTwo = null; // code here!
  *         (1) causes the odometer in the object to be increased by the distance,
  *         (2) returns the updated value of the `odometer`.
 */
-function carMaker(/* code here */) {
-  /* code here */
+
+function carMaker(odometer) {
+    
+  return {
+    odometer: odometer,
+    drive:function(distance) { 
+     return  this.odometer = distance + this.odometer;
+    }
+  }
+
 }
```

Comment:
Great JOB! using the `this` keyword.  understanding the `this` keyword will really help when we get to the react modules.